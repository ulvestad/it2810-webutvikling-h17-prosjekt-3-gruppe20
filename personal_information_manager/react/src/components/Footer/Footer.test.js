import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Footer from './Footer';

const jestExpect = global.expect;

it('Footer renders without crashing', () => {
  shallow(<Footer />);
});

it('renders correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  jestExpect(tree).toMatchSnapshot();
});
