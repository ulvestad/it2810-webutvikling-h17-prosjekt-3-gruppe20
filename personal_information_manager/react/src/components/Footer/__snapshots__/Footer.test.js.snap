// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="py-5 bg-dark"
>
  <div
    className="container"
  >
    <p
      className="m-0 text-center text-white"
    >
      Copyright Â© Gruppe 20 - IT2810@NTNU
    </p>
  </div>
</footer>
`;
